### DISCUSSION TITLE
How to disable auto linefeed after typing SELECT

### DISCUSSION THREAD

**Discussion by weaver543** [dbeaver/dbeaver#18494] [General]
If the following line contains a just a space (and maybe other stuff, not sure), then when I start to type SELECT and let it autocomplete, a linefeed is automatically included in the autocompleted SELECT, leaving my cursor on the following line, so I have to backspace back to the previous line. 

how can I disable this behavior?

(latest Mac version 22.3.0)

**Comment by LonwoLonwo** [comment]
Hello @weaver543 

Can you please provide a gif or a video of this issue?
